### PR TITLE
windows/* : add Tamil translation

### PR DESCRIPTION
**Pull request by ndrohith09**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Thanks for this massive PR, Welcome to tldr. I will review it later today.

> இந்த பெரிய PRக்கு நன்றி, tldrக்கு வரவேற்கிறோம். நான் இதை இன்று பின்னர் மதிப்பாய்வு செய்கிறேன்.

**Comment by ndrohith09** [comment]
Thank you @kbdharun

File: pages.ta/windows/choco-install.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`choco install {{path/to/packages.config}}`
+
+- ஒரு குறிப்பிட்ட nuspec அல்லது nupkg கோப்பை நிறுவவும்:
+
+`choco install {{path/to/file}}`
+
+- தொகுப்பின் குறிப்பிட்ட பதிப்பை நிறுவவும்:
+
+`choco install {{package}} --version {{version}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-install.md:20]:
```suggestion
`choco install {{நிரல்தொகுப்பு}} --version {{பதிப்பு}}`
```

File: pages.ta/windows/choco-install.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+> சாக்லேட்டியுடன் ஒன்று அல்லது அதற்கு மேற்பட்ட தொகுப்புகளை நிறுவவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-install>.
+
+- ஒன்று அல்லது அதற்கு மேற்பட்ட இடம் பிரிக்கப்பட்ட தொகுப்புகளை நிறுவவும்:
+
+`choco install {{package(s)}}`
+
+- தனிப்பயன் உள்ளமைவு கோப்பிலிருந்து தொகுப்புகளை நிறுவவும்:
+
+`choco install {{path/to/packages.config}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-install.md:12]:
```suggestion
`choco install {{பாதை/டு/நிரல்தொகுப்பு.config}}`
```

File: pages.ta/windows/choco-install.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- ஒரு குறிப்பிட்ட nuspec அல்லது nupkg கோப்பை நிறுவவும்:
+
+`choco install {{path/to/file}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-install.md:16]:
```suggestion
`choco install {{பாதை/டு/கோப்பு}}`
```

File: pages.ta/windows/choco-install.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- ஒன்று அல்லது அதற்கு மேற்பட்ட இடம் பிரிக்கப்பட்ட தொகுப்புகளை நிறுவவும்:
+
+`choco install {{package(s)}}`
+
+- தனிப்பயன் உள்ளமைவு கோப்பிலிருந்து தொகுப்புகளை நிறுவவும்:
+
+`choco install {{path/to/packages.config}}`
+
+- ஒரு குறிப்பிட்ட nuspec அல்லது nupkg கோப்பை நிறுவவும்:
```
Comment (kbdharun) [inline pages.ta/windows/choco-install.md:14]:
```suggestion
- ஒரு குறிப்பிட்ட `nuspec` அல்லது `nupkg` கோப்பை நிறுவவும்:
```
Even though this isn't present on the English page, I prefer this in translations.

File: pages.ta/windows/choco-list.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# choco list
+
+> சாக்லேட்டியுடன் ஒன்று அல்லது அதற்கு மேற்பட்ட தொகுப்புகளை நிறுவவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-install>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-list.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-install>.
```

File: pages.ta/windows/choco-list.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`choco list --approved-only`
+
+- இதிலிருந்து தொகுப்புகளைக் காண்பிக்க தனிப்பயன் மூலத்தைக் குறிப்பிடவும்:
+
+`choco list --source {{source_url|alias}}`
+
+- அங்கீகாரத்திற்கான பயனர்பெயர் மற்றும் கடவுச்சொல்லை வழங்கவும்:
+
+`choco list --user {{username}} --password {{password}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-list.md:28]:
```suggestion
`choco list --user {{பயனர்பெயர்}} --password {{கடவுச்சொல்}}`
```

File: pages.ta/windows/choco-new.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# choco new
+
+> சாக்லேட்டியுடன் புதிய தொகுப்பு விவரக்குறிப்பு கோப்புகளை உருவாக்கவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-new>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-new>.
```

File: pages.ta/windows/choco-new.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- ஒரு புதிய தொகுப்பு எலும்புக்கூட்டை உருவாக்கவும்:
+
+`choco new {{package_name}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:8]:
```suggestion
`choco new {{நிரல்தொகுப்பு_பெயர்}}`
```

File: pages.ta/windows/choco-new.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`choco new {{package_name}} --maintainer {{maintainer_name}}`
+
+- தனிப்பயன் வெளியீட்டு கோப்பகத்தில் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --output-directory {{path/to/directory}}`
+
+- குறிப்பிட்ட 32-பிட் மற்றும் 64-பிட் நிறுவி URLகளுடன் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco புதிய {{package_name}} url="{{url}}" url64="{{url}}"`
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:24]:
```suggestion
`choco new {{நிரல்தொகுப்பு_பெயர்}} url="{{url}}" url64="{{url}}"`
```
I think you accidentally translated the commands too.

File: pages.ta/windows/choco-new.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+`choco new {{package_name}} --version {{version}}`
+
+- குறிப்பிட்ட பராமரிப்பாளர் பெயருடன் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --maintainer {{maintainer_name}}`
+
+- தனிப்பயன் வெளியீட்டு கோப்பகத்தில் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --output-directory {{path/to/directory}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:20]:
```suggestion
`choco new {{நிரல்தொகுப்பு_பெயர்}} --output-directory {{அடைவிற்குப்/பாதை}}`
```

File: pages.ta/windows/choco-new.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`choco new {{package_name}}`
+
+- ஒரு குறிப்பிட்ட பதிப்பில் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --version {{version}}`
+
+- குறிப்பிட்ட பராமரிப்பாளர் பெயருடன் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --maintainer {{maintainer_name}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:16]:
```suggestion
`choco new {{நிரல்தொகுப்பு_பெயர்}} --maintainer {{பராமரிப்பாளர்_பெயர்}}`
```

File: pages.ta/windows/choco-new.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+> சாக்லேட்டியுடன் புதிய தொகுப்பு விவரக்குறிப்பு கோப்புகளை உருவாக்கவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-new>.
+
+- ஒரு புதிய தொகுப்பு எலும்புக்கூட்டை உருவாக்கவும்:
+
+`choco new {{package_name}}`
+
+- ஒரு குறிப்பிட்ட பதிப்பில் புதிய தொகுப்பை உருவாக்கவும்:
+
+`choco new {{package_name}} --version {{version}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-new.md:12]:
```suggestion
`choco new {{நிரல்தொகுப்பு_பெயர்}} --version {{பதிப்பு}}`
```

File: pages.ta/windows/choco-outdated.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# choco outdated
+
+> சாக்லேட்டியுடன் காலாவதியான பேக்கேஜ்களைச் சரிபார்க்கவும்.
+> மேலும் தகவல்:  <https://chocolatey.org/docs/commands-outdated>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-outdated.md:4]:
```suggestion
> மேலும் விவரத்திற்கு:  <https://chocolatey.org/docs/commands-outdated>.
```

File: pages.ta/windows/choco-outdated.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+# choco outdated
+
+> சாக்லேட்டியுடன் காலாவதியான பேக்கேஜ்களைச் சரிபார்க்கவும்.
```
Comment (kbdharun) [inline pages.ta/windows/choco-outdated.md:3]:
```suggestion
> சாக்லேட்டியுடன் காலாவதியான தொகுப்புகளைச் சரிபார்க்கவும்.
```

File: pages.ta/windows/choco-outdated.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+`choco outdated --ignore-pinned`
+
+- தொகுப்புகளை சரிபார்க்க தனிப்பயன் மூலத்தைக் குறிப்பிடவும்:
+
+`choco outdated --source {{source_url|alias}}`
+
+- அங்கீகாரத்திற்கான பயனர்பெயர் மற்றும் கடவுச்சொல்லை வழங்கவும்:
+
+`choco outdated --user {{username}} --password {{password}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-outdated.md:20]:
```suggestion
`choco outdated --user {{பயனர்பெயர்}} --password {{கடவுச்சொல்}}`
```

File: pages.ta/windows/choco-pack.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
+# choco pack
+
+> ஒரு NuGet விவரக்குறிப்பை nupkg கோப்பில் தொகுக்கவும்.
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:3]:
```suggestion
> ஒரு `NuGet` விவரக்குறிப்பை `nupkg` கோப்பில் தொகுக்கவும்.
```

File: pages.ta/windows/choco-pack.md [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-pack>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-pack>.
```

File: pages.ta/windows/choco-pack.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+
+- ஒரு nupkg கோப்பில் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:6]:
```suggestion
- ஒரு `nupkg` கோப்பில் ஒரு `NuGet` விவரக்குறிப்பைத் தொகுக்கவும்:
```

File: pages.ta/windows/choco-pack.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+
+`choco pack {{path/to/specification}}`
+
+- இதன் விளைவாக வரும் கோப்பின் பதிப்பைக் குறிப்பிடும் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:10]:
```suggestion
- இதன் விளைவாக வரும் கோப்பின் பதிப்பைக் குறிப்பிடும் ஒரு `NuGet` விவரக்குறிப்பைத் தொகுக்கவும்:
```

File: pages.ta/windows/choco-pack.md [hunk 20]
```diff
@@ -0,0 +1,16 @@
+# choco pack
+
+> ஒரு NuGet விவரக்குறிப்பை nupkg கோப்பில் தொகுக்கவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-pack>.
+
+- ஒரு nupkg கோப்பில் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
+
+`choco pack {{path/to/specification}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:8]:
```suggestion
`choco pack {{பாதை/டு/விவரக்குறிப்பு}}`
```

File: pages.ta/windows/choco-pack.md [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+
+- இதன் விளைவாக வரும் கோப்பின் பதிப்பைக் குறிப்பிடும் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
+
+`choco pack {{path/to/specification}} --version {{version}}`
+
+- ஒரு குறிப்பிட்ட கோப்பகத்திற்கு ஒரு NuGet விவரக்குறிப்பை தொகுக்கவும்:
+
+`choco pack {{path/to/specification}} --output-directory {{path/to/output_directory}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:16]:
```suggestion
`choco pack {{பாதை/டு/விவரக்குறிப்பு}} --output-directory {{பாதை/டு/வெளியீடு_கோப்பகம்}}`
```

File: pages.ta/windows/choco-pack.md [hunk 22]
```diff
@@ -0,0 +1,16 @@
...
+
+- ஒரு nupkg கோப்பில் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
+
+`choco pack {{path/to/specification}}`
+
+- இதன் விளைவாக வரும் கோப்பின் பதிப்பைக் குறிப்பிடும் ஒரு NuGet விவரக்குறிப்பைத் தொகுக்கவும்:
+
+`choco pack {{path/to/specification}} --version {{version}}`
+
+- ஒரு குறிப்பிட்ட கோப்பகத்திற்கு ஒரு NuGet விவரக்குறிப்பை தொகுக்கவும்:
```
Comment (kbdharun) [inline pages.ta/windows/choco-pack.md:14]:
```suggestion
- ஒரு குறிப்பிட்ட கோப்பகத்திற்கு ஒரு `NuGet` விவரக்குறிப்பை தொகுக்கவும்:
```

File: pages.ta/windows/choco-source.md [hunk 23]
```diff
@@ -0,0 +1,32 @@
+# choco source
+
+> சாக்லேட் மூலம் தொகுப்புகளுக்கான ஆதாரங்களை நிர்வகிக்கவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-source>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-source>.
```

File: pages.ta/windows/choco-source.md [hunk 24]
```diff
@@ -0,0 +1,32 @@
+# choco source
+
+> சாக்லேட் மூலம் தொகுப்புகளுக்கான ஆதாரங்களை நிர்வகிக்கவும்.
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:3]:
```suggestion
> சாக்லேட்டி மூலம் தொகுப்புகளுக்கான ஆதாரங்களை நிர்வகிக்கவும்.
```

File: pages.ta/windows/choco-source.md [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-source>.
+
+- தற்போது கிடைக்கக்கூடிய ஆதாரங்களை பட்டியலிடுங்கள்:
+
+`choco source list`
+
+- புதிய தொகுப்பு மூலத்தைச் சேர்க்கவும்:
+
+`choco source add --name {{name}} --source {{url}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:12]:
```suggestion
`choco source add --name {{பெயர்}} --source {{url}}`
```

File: pages.ta/windows/choco-source.md [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+
+`choco source add --name {{name}} --source {{url}} --user {{username}} --password {{password}}`
+
+- கிளையன்ட் சான்றிதழுடன் புதிய தொகுப்பு மூலத்தைச் சேர்க்கவும்:
+
+`choco source add --name {{name}} --source {{url}} --cert {{path/to/certificate}}`
+
+- தொகுப்பு மூலத்தை இயக்கு:
+
+`choco source enable --name {{name}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:24]:
```suggestion
`choco source enable --name {{பெயர்}}`
```

File: pages.ta/windows/choco-source.md [hunk 27]
```diff
@@ -0,0 +1,32 @@
...
+
+- ஒரு தொகுப்பு மூலத்தை முடக்கு:
+
+`choco source disable --name {{name}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:28]:
```suggestion
`choco source disable --name {{பெயர்}}`
```

File: pages.ta/windows/choco-source.md [hunk 28]
```diff
@@ -0,0 +1,32 @@
...
+
+- தொகுப்பு மூலத்தை அகற்றவும்:
+
+`choco source remove --name {{name}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:32]:
```suggestion
`choco source remove --name {{பெயர்}}`
```

File: pages.ta/windows/choco-source.md [hunk 29]
```diff
@@ -0,0 +1,32 @@
...
+
+`choco source list`
+
+- புதிய தொகுப்பு மூலத்தைச் சேர்க்கவும்:
+
+`choco source add --name {{name}} --source {{url}}`
+
+- நற்சான்றிதழ்களுடன் புதிய தொகுப்பு மூலத்தைச் சேர்க்கவும்:
+
+`choco source add --name {{name}} --source {{url}} --user {{username}} --password {{password}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:16]:
```suggestion
`choco source add --name {{பெயர்}} --source {{url}} --user {{பயனர்பெயர்}} --password {{கடவுச்சொல்}}`
```

File: pages.ta/windows/choco-source.md [hunk 30]
```diff
@@ -0,0 +1,32 @@
...
+
+- கிளையன்ட் சான்றிதழுடன் புதிய தொகுப்பு மூலத்தைச் சேர்க்கவும்:
+
+`choco source add --name {{name}} --source {{url}} --cert {{path/to/certificate}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-source.md:20]:
```suggestion
`choco source add --name {{பெயர்}} --source {{url}} --cert {{பாதை/டு/சான்றிதழ்}}`
```

File: pages.ta/windows/choco-pin.md [hunk 31]
```diff
@@ -0,0 +1,21 @@
+# choco pin
+
+> சாக்லேட்டியுடன் ஒரு குறிப்பிட்ட பதிப்பில் ஒரு தொகுப்பைப் பின் செய்யவும்.
+> மேம்படுத்தும் போது பின் செய்யப்பட்ட தொகுப்புகள் தானாகவே தவிர்க்கப்படும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-pin>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-pin.md:5]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-pin>.
```

File: pages.ta/windows/choco-pin.md [hunk 32]
```diff
@@ -0,0 +1,21 @@
...
+
+- பின் செய்யப்பட்ட தொகுப்புகள் மற்றும் அவற்றின் பதிப்புகளின் பட்டியலைக் காண்பி:
+
+`choco pin list`
+
+- ஒரு தொகுப்பை அதன் தற்போதைய பதிப்பில் பின் செய்யவும்:
+
+`choco pin add --name {{package}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-pin.md:13]:
```suggestion
`choco pin add --name {{நிரல்தொகுப்பு}}`
```

File: pages.ta/windows/choco-pin.md [hunk 33]
```diff
@@ -0,0 +1,21 @@
...
+
+- ஒரு குறிப்பிட்ட பதிப்பில் ஒரு தொகுப்பைப் பின் செய்யவும்:
+
+`choco pin add --name {{package}} --version {{version}}`
+
+- ஒரு குறிப்பிட்ட தொகுப்பிற்கான பின்னை அகற்றவும்:
+
+`choco pin remove --name {{package}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-pin.md:21]:
```suggestion
`choco pin remove --name {{நிரல்தொகுப்பு}}`
```

File: pages.ta/windows/choco-pin.md [hunk 34]
```diff
@@ -0,0 +1,21 @@
...
+
+`choco pin list`
+
+- ஒரு தொகுப்பை அதன் தற்போதைய பதிப்பில் பின் செய்யவும்:
+
+`choco pin add --name {{package}}`
+
+- ஒரு குறிப்பிட்ட பதிப்பில் ஒரு தொகுப்பைப் பின் செய்யவும்:
+
+`choco pin add --name {{package}} --version {{version}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-pin.md:17]:
```suggestion
`choco pin add --name {{நிரல்தொகுப்பு}} --version {{பதிப்பு}}`
```

File: pages.ta/windows/choco-search.md [hunk 35]
```diff
@@ -0,0 +1,28 @@
+# choco search
+
+> சாக்லேட்டியுடன் உள்ளூர் அல்லது தொலைநிலைப் பொதியைத் தேடுங்கள்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-search>.
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:4]:
```suggestion
> மேலும் விவரத்திற்கு: <https://chocolatey.org/docs/commands-search>.
```

File: pages.ta/windows/choco-search.md [hunk 36]
```diff
@@ -0,0 +1,28 @@
...
+
+- ஒரு தொகுப்பைத் தேடுங்கள்:
+
+`choco search {{query}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:8]:
```suggestion
`choco search {{வினவல்}}`
```

File: pages.ta/windows/choco-search.md [hunk 37]
```diff
@@ -0,0 +1,28 @@
...
+
+- உள்நாட்டில் ஒரு தொகுப்பைத் தேடுங்கள்:
+
+`choco search {{query}} --local-only`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:12]:
```suggestion
`choco search {{வினவல்}} --local-only`
```

File: pages.ta/windows/choco-search.md [hunk 38]
```diff
@@ -0,0 +1,28 @@
...
+
+- முடிவுகளில் சரியான பொருத்தங்களை மட்டும் சேர்க்கவும்:
+
+`choco search {{query}} --exact`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:16]:
```suggestion
`choco search {{வினவல்}} --exact`
```

File: pages.ta/windows/choco-search.md [hunk 39]
```diff
@@ -0,0 +1,28 @@
...
+
+- அனைத்து அறிவுறுத்தல்களையும் தானாக உறுதிப்படுத்தவும்:
+
+`choco search {{query}} --yes`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:20]:
```suggestion
`choco search {{வினவல்}} --yes`
```

File: pages.ta/windows/choco-search.md [hunk 40]
```diff
@@ -0,0 +1,28 @@
...
+
+- தொகுப்புகளைத் தேட தனிப்பயன் மூலத்தைக் குறிப்பிடவும்:
+
+`choco search {{query}} --source {{source_url|alias}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:24]:
```suggestion
`choco search {{வினவல்}} --source {{source_url|alias}}`
```

File: pages.ta/windows/choco-search.md [hunk 41]
```diff
@@ -0,0 +1,28 @@
...
+
+- அங்கீகாரத்திற்கான பயனர்பெயர் மற்றும் கடவுச்சொல்லை வழங்கவும்:
+
+`choco search {{query}} --user {{username}} --password {{password}}`
```
Comment (kbdharun) [inline pages.ta/windows/choco-search.md:28]:
```suggestion
`choco search {{வினவல்}} --user {{பயனர்ப்பெயர்}} --password {{கடவுச்சொல்}}`
```

**Comment by kbdharun** [comment]
Thank you so much for this PR. Keep up the good work. It's not every day that I get to review Tamil translations.

> Usually I would request changes from translators or PR authors. But since I am a native speaker, I took up the work and proofread this PR and applied our usual suggestions. Sorry if I overwhelmed you with 157 review comments.

- **Please batch all my suggestions in the `Files changed` tab and make a single commit in this PR.**

I will make some suggestions that will be useful for you in future, 
- Refer to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) in future for paths.
- Translate command arguments inside `{{}}` in future as it will be easy for native speakers to understand the placeholders/arguments.
- Try to make a distinct difference between a command name and description use `` whenever necessary.
- For terms like API, etc don't translate them to Tamil. Except in some cases in placeholders/command arguments.

I guess I covered it all.

You might ask me, why we use certain words or follow certain methods in some cases.

The answer is that
- For words like packages `தொகுப்பு` is correct but we prefer `நிரல்தொகுப்பு` (program package) as it provides a distinction to the native speaker between a package used in computer (software) and our physical parcels/packages.
- For "More information:" links we use `மேலும் விவரத்திற்கு:` instead of `மேலும் தகவல்:` even though it is also right because it works better with all our pages. This has been discussed in the past and we use this term after reaching a consensus. You can refer to this [here](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py).
- Paths is the only place where we struggle, if you noticed my suggestions we used `அடைவிற்குப்/பாதை` in most places for `path/to/directory`, it is grammatically correct but it can't be applied in all pages (paths, when translated, might confuse the readers) and for `path/to/file`, I suggested `பாதை/டு/கோப்பு` instead of `கோப்பின்/பாதை ` because the later which is grammatically correct can't be used in all places. So we are using `டு` which is a direct transliteration of the English word `to`. [Based on different use cases we use a different format of paths].
- For alias pages, we generally use the one present in this [file](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md), but it isn't standardised except the first line, so we can change the alias pages content or format to our flexibility based on usage.

Seems like I wrote an essay. 
Time for big news, I am working on a separate style guide for Tamil I will probably add it by this Diwali time.

Once again, Thanks.
Regards,
Dharun Krishna

**Comment by ndrohith09** [comment]
Not a problem @kbdharun  . I learnt a lot . Will make the necessary changes and will commit .

**Comment by ndrohith09** [comment]
@kbdharun I have made all the changes which was specified by you .. Thanks a lot for helping me . I learnt many things from this one PR . Wishing to learn a lot from you .

File: pages.ta/windows/azcopy.md [hunk 42]
```diff
@@ -0,0 +1,28 @@
...
+
+`azcopy copy '{{பாதை/டு/மூலம்}}' 'https://{{சேமிப்பு_கணக்கு_பெயர்}}.blob.core.windows.net/{{கொள்கலன்_பெயர்}}' --include-pattern '{{*.txt;*.jpg}}'`
+
+- இரண்டு அசூர் சேமிப்பு கணக்குகளுக்கு இடையே நேரடியாக ஒரு கொள்கலனை நகலெடுக்கவும்:
+
+`azcopy copy 'https://{{மூலம்_சேமிப்பு_கணக்கு_பெயர்}}.blob.core.windows.net/{{கொள்கலன்_பெயர்}}' 'https://{{சேருமிடம்_சேமிப்பு_கணக்கு_பெயர்}}.blob.core.windows.net/{{கொள்கலன்_பெயர்}}'`
+
+- ஒரு உள்ளூர் கோப்பகத்தை ஒத்திசைக்கவும், மேலும் மூலத்தில் கோப்புகள் இல்லை என்றால் இலக்கில் உள்ள கோப்புகளை நீக்கவும்:
+
+`azcopy sync '{{path/to/source}}' 'https://{{storage_account_name}}.blob.core.windows.net/{{container_name}}' --recursive --delete-destination=true`
```
Comment (kbdharun) [inline pages.ta/windows/azcopy.md:24]:
```suggestion
`azcopy sync '{{பாதை/டு/மூலம்}}' 'https://{{சேமிப்பு_கணக்கு_பெயர்}}.blob.core.windows.net/{{கொள்கலன்_பெயர்}}' --recursive --delete-destination=true`
```

File: pages.ta/windows/comp.md [hunk 43]
```diff
@@ -0,0 +1,37 @@
...
+
+`comp /a {{பாதை/டு/கோப்பு_1}} {{பாதை/டு/கோப்பு_2}}`
+
+- வேறுபாடுகளுக்கான வரி எண்களைக் காண்பி:
+
+`comp /l {{path/to/file_1}} {{பாதை/டு/கோப்பு_2}}`
+
+- கோப்புகளை கேஸ்-உணர்வின்றி ஒப்பிடுக:
+
+`comp /c {{path/to/file_1}} {{பாதை/டு/கோப்பு_2}}`
```
Comment (kbdharun) [inline pages.ta/windows/comp.md:33]:
```suggestion
`comp /c {{பாதை/டு/கோப்பு_1}} {{பாதை/டு/கோப்பு_2}}`
```

File: pages.ta/windows/comp.md [hunk 44]
```diff
@@ -0,0 +1,37 @@
...
+
+- ஒவ்வொரு கோப்பின் முதல் 5 வரிகளை மட்டும் ஒப்பிடுக:
+
+`comp /n={{5}} {{path/to/file_1}} {{பாதை/டு/கோப்பு_2}}`
```
Comment (kbdharun) [inline pages.ta/windows/comp.md:37]:
```suggestion
`comp /n={{5}} {{பாதை/டு/கோப்பு_1}} {{பாதை/டு/கோப்பு_2}}`
```

**Review by kbdharun — COMMENTED** [review]
My final 3 suggestions with this applied the PR is good to go.

**Comment by ndrohith09** [comment]
@kbdharun The concerned changes were done.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**